From 86c7842a7723258ef1016ff44785f66b16668bd1 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Mon, 19 Oct 2020 18:59:36 +0000
Subject: [PATCH] Don't build crossgen2 self-contained

When self-contained is true, Crossgen2 attemts to download
Microsoft.NETCore.App.Runtime.linux-x64 with a version matching
the SDK version of the SDK from global.json
---
 .../projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj b/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj
index 7ae30c5..db13cdd 100644
--- a/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj
+++ b/src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Crossgen2.pkgproj
@@ -68,7 +68,7 @@
             Targets="Restore"
             Properties="
             Configuration=$(RuntimeConfiguration);
-            SelfContained=true;
+            SelfContained=false;
             RuntimeIdentifier=$(PackageRID)">
     <Output TaskParameter="TargetOutputs" ItemName="Crossgen2File" />
     </MSBuild>
@@ -79,7 +79,7 @@
             Configuration=$(RuntimeConfiguration);
             GenerateDependencyFile=true;
             IncludeFileVersionsInDependencyFile=false;
-            SelfContained=true;
+            SelfContained=false;
             RemoveLongNameDac=true;
             RuntimeGraphPath=$(LiveRuntimeIdentifierGraphPath);
             RuntimeIdentifier=$(PackageRID)">
@@ -90,7 +90,7 @@
       <File Include="@(Crossgen2File)">
         <TargetPath>tools</TargetPath>
       </File>
-      <!-- 
+      <!--
         System.Private.CoreLib's R2R pdb ends up in two locations (CoreCLR and the depproj output).
         Remove them here and let the symbol file location infra find exactly one.
       -->
-- 
1.8.3.1

